low-level/framework/messaging/egress_processor.py

```diff
@@ -209,17 +209,19 @@ def _transmit_msg_on_exchange(self):
                 logger.debug(
                     "_transmit_msg_on_exchange, Successfully Sent: %s" % self._jsonMsg)
 
+            # DEPRICATED: LoggingMsgHandler is no more in use. IEM routing
+            #             is handled by IEM() class and other MsgHandlers.
             # Routing requests for IEM msgs sent from the LoggingMsgHandler
-            elif self._jsonMsg.get("message").get("IEM_routing") is not None:
-                log_msg = self._jsonMsg.get("message").get("IEM_routing").get(
-                    "log_msg")
-                if self._iem_route_addr != "":
-                    self._producer.send([json.dumps(self._jsonMsg)])
-                else:
-                    logger.warn(
-                        "EgressProcessor, Attempted to route IEM without a valid 'iem_route_addr' set.")
-                logger.debug(
-                    "_transmit_msg_on_exchange, Successfully Sent: %s" % log_msg)
+            # elif self._jsonMsg.get("message").get("IEM_routing") is not None:
```

Comment:
actually remove old code. These references we can have from cortx-1.0 branch or git history also

Reply:
done 👍